api/src/main/java/edu/wpi/first/shuffleboard/api/LiveWindow.java

```diff
@@ -12,14 +12,15 @@
  */
 public final class LiveWindow {
 
-  private static final ITable liveWindowTable = NetworkTable.getTable("LiveWindow");
+  private static final NetworkTableInstance inst = NetworkTableInstance.getDefault();
+  private static final NetworkTable liveWindowTable = inst.getTable("LiveWindow");
 
   private static final BooleanProperty enabled = new SimpleBooleanProperty(LiveWindow.class, "enabled", false);
 
   static {
-    liveWindowTable.getSubTable("~STATUS~").addTableListenerEx("LW Enabled", (source, key, value, isNew) -> {
-      if (value instanceof Boolean) {
-        enabled.set((Boolean) value);
+    liveWindowTable.getSubTable("~STATUS~").addEntryListener("LW Enabled", (table, source, key, value, flags) -> {
```

Comment:
Double underscore for `table` if it's not used?

Reply:
I don't think that's necessary. You'd have to do the same for `source`, `key`, `flags`, and we'd have to wait for Java 10 to let us use a single underscore for multiple unused lambda parameters